pkg/testing/ess/users.go

```diff
@@ -0,0 +1,37 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package ess
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
+type GetUserRequest struct {
+	// For future use
+}
+
+type GetUserResponse struct {
+	User struct {
+		UserID int `json:"user_id"`
+	} `json:"user"`
+}
+
+// GetUser returns information about the authenticated user
+func (c *Client) GetUser(ctx context.Context, req GetUserRequest) (*GetUserResponse, error) {
```

Comment:
Nitpicking: as a preference I like to see the methods close to the types that they refer to... Why is this here and not in `client.go` ?

Reply:
Mostly because the ESS API is quite large.  I don't think we'd every fully implement it but, if we did, `client.go` might get quite large.

Nevertheless, your point is well taken.  We should start by keeping everything in `client.go`.  If it starts to get really large, we can consider breaking it up.

Reply:
I like the idea of keeping each area of the API in its own file, so please leave it like it is.